hat/assets/js/apps/Iaso/domains/payments/types.ts

```diff
@@ -36,6 +44,13 @@ export type Payment = {
     change_requests: OrgUnitChangeRequest[];
 };
 
+type NestedPayment = {
+    id: number;
+    status: PaymenStatus;
+    user: User;
+    change_requests: OrgUnitChangeRequest[];
+};
```

Comment:
```suggestion
type NestedPayment = Omit<Payment,"created_at", "updated_at","created_by","updated_by">
```

`Omit` will need to be imported